src/annotator/plugin/document.js

```diff
@@ -177,10 +177,10 @@ export default class DocumentMeta extends Delegator {
     /** @type {Object.<string,string[]>} */
     const tags = {};
     for (let meta of Array.from(this.document.querySelectorAll('meta'))) {
-      const name = meta.getAttribute(attribute);
+      const name = meta.getAttribute(attribute)?.toLowerCase();
```

Comment:
This is going to affect the metadata that is stored with the annotation and sent via the API when it is created. I suspect it is probably fine to always lower case the name value and not store the original version. It would be worth checking whether the backend references any of these values explicitly and whether it does so in a case-insensitive way.

Reply:
I think I am going to need a little help to check the backend code.